internal/provider/resource_device.go

```diff
@@ -372,6 +379,7 @@ func toPortOverride(data map[string]interface{}) (unifi.DevicePortOverrides, err
 		OpMode:            opMode,
 		PoeMode:           poeMode,
 		AggregateNumPorts: aggregateNumPorts,
+		NATiveNetworkID:   nativeNetworkID,
```

Comment:
```suggestion
		NativeNetworkID:   nativeNetworkID,
```

NATiveNetworkID looks like it was named incorrectly, is it intentional?